### PR TITLE
added basic of classes and objects in python

### PR DESCRIPTION
**Pull request by Khanna14Aman**
### Describe your change

- [x] Add a concept ?
- [ ] Fix a bug or typo in an existing concept ?
- [ ] Documentation change?

### Checklist

- [x] I have read [CONTRIBUTING.md](https://github.com/dev-madhurendra/OOPs/blob/main/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one file. To ease review, please open separate PRs for separate concepts.
- [x] All new folders are placed in an existing directory.
- [x] All new files are placed inside an existing directory.
- [x] All folder names should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`Golang` is allowed but `golang` is not
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`Class.cpp` is allowed but `classe.cpp` is not
- [x] All new concepts have a URL in their comments that points to Wikipedia or another similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`

### CODE REVIEW

**Review by dev-madhurendra — APPROVED** [review]

File: Python/Class/Class.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+'''
+    1- Class is a user-define data type. It consist of data members and member function.
+    2- Class gives the blueprint structure that how the memory will get allocated when object of class get created.
+    3- In python for creating class we use keyword "class" along with the name of class what you want.
+    1- Object is a real world entity which used to access class functionality.
+    2- Memory gets allocated only when object get created.
+'''
+
+
+# Here we define a new class named "First"
```
Comment (dev-madhurendra) [inline Python/Class/Class.py:10]: But here class name is Bike you written First

**Review by dev-madhurendra — CHANGES_REQUESTED** [review]

**Comment by dev-madhurendra** [comment]
Hi @Khanna14Aman , I am approving this PR but follow the naming convention of branch,file,folder name e.g. branch name for this should be : Python_Class

**Review by dev-madhurendra — APPROVED** [review]
Also write proper comment for the concepts so that anyone can understand easily by just reading.

**Review by Suryac72 — APPROVED** [review]
Good from my end, handover to @dev-madhurendra

**Pull request merged**